sympy/physics/continuum_mechanics/beam.py

```diff
@@ -3332,3 +3332,229 @@ def plot_shear_stress(self, dir="all", subs=None):
             Py = self._plot_shear_stress('y', subs)
             Pz = self._plot_shear_stress('z', subs)
             return PlotGrid(3, 1, Px, Py, Pz)
+
+    def max_shear_force(self, dir=None):
```

Comment:
The default direction should be `x`, similar to what has been previously done in beam3d

Reply:
Also, why are we making these functions direction specific? It would be better to calculate the max_points for all directions and return a list of tuple.